src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -13,6 +13,7 @@ export default () => `
 	<div class="welcomePage">
 		<div class="ads-homepage splash">
 			<div class="gradient">
+				<div class="content-container">
```

Comment:
nit: please indent the inner content properly to clearly be able to tell what is contained within this div